### ISSUE TITLE
Could you share the Inception model with classification heads version and the checkpoint plz?

### ISSUE THREAD

**Issue by nietzsche9088** [matthias-wright/art-fid#16]
Hi, I'm testing your model's performance in the classification task on WikiArt. But it doesn't work well with my classification head. My model is like guessing. 

In this following code, I load the CKPT you provide for the **inception** input. 
`
class ClassificationHead(nn.Module):
    def __init__(self, num_labels):
        super().__init__()
        self.num_labels = num_labels

    def forward(self, x):
        linear = nn.Linear(x.shape[1], self.num_labels).to(x.device)
        x = linear(x)
        x = F.log_softmax(x, dim=1)   
        return x
`
    
`
class Model(nn.Module):
    def __init__(self, inception, num_style_classes=10):
        super().__init__()
        self.encoder = inception
        self.style_head = ClassificationHead(num_style_classes)

    def forward(self, x):
        feat = self.encoder(x, return_features=True)
        feat = feat.view(feat.size(0), -1)
        style_logits = self.style_head(feat)

        return style_logits
`
Thanks in advance.
